PWGHF/vertexingHF/AliHFSystErr.cxx

```diff
@@ -7019,6 +7020,81 @@ void AliHFSystErr::InitLctopK0S2016pPb() {
 
 }
 
+void AliHFSystErr::InitLctopK0S2015pp() {
+  //
+  // Lc->pK0S syst errors. Responsible: A. De Caro, E. Meninno
+  //  2016 p-Pb sample
+  //
+  SetNameTitle("AliHFSystErr","SystErrLctopK0S2013pPb");
+
```

Comment:
Which data sample are these uncertainties referring to?
From the name of the method it seems they are from 2015 pp (13 TeV? or 5 TeV?)
Then the commented lines refer to 2016 p-Pb, and the SetNameTitle sets 2013pPb.
I suggest to clarify this, to avoid confusion in the future when we will check this code